zppy/amwg.py

```diff
@@ -49,10 +49,11 @@ def amwg(config, scriptDir):
             # List of dependencies
             dependencies = [ os.path.join(scriptDir, 'climo_%s_%04d-%04d.status' % (c['grid'],c['year1'],c['year2'])), ]
 
-            # Submit job
-            jobid = submitScript(scriptFile, dependFiles=dependencies, export='NONE')
+            if not c['dry_run']:
```

Comment:
This file is never tested with `active = True`.

Reply:
Yes, this is a user-contributed diagnostics.